installer/windows/invest_installer.nsi

```diff
@@ -451,8 +451,8 @@ Section "InVEST Tools" Section_InVEST_Tools
         !insertmacro StartMenuLink "${SMPATH}\DelineateIt" "delineateit"
         !insertmacro StartMenuLink "${SMPATH}\Visitation: Recreation and Tourism" "recreation"
         !insertmacro StartMenuLink "${SMPATH}\Urban Flood Risk Mitigation" "ufrm"
-        !insertmacro StartMenuLink "${SMPATH}\Urban Cooling Model" "ucm"
-        !insertmacro StartMenuLink "${SMPATH}\Urban Stormwater Retention Model" "stormwater"
+        !insertmacro StartMenuLink "${SMPATH}\Urban Cooling" "ucm"
```

Comment:
to be consistent with the names in `natcap/invest/__init__.py`, leaving out "Model"